ui/pages/home/index.scss

```diff
@@ -279,8 +279,12 @@
     }
   }
 
-  &__survey-banner {
-    position: absolute;
+  &__overlay-banners {
+    z-index: 9999;
```

Comment:
Why are we using this z-index?

Reply:
I found that the current implementation was being shown under other elements in the DOM.

The toasts should always be on top of everything, so the best way to ensure this is with z-index, as there are currently other elements using z-index as well.